### DISCUSSION TITLE
Frequently Usage

### DISCUSSION THREAD

**Discussion by kihanb** [norkunas/youtube-dl-php#188] [General]
Hello Guys,
I'm working on a project that will be used for downloading videos from youtube and other websites but I think there is such a limit for downloading videos from youtube.
I mean If this script download many videos with same ip address, will the IP be restricted or blocked?

shall I use some proxies like tor or stuff?

**Comment by norkunas** [comment]
Hi,
If you think that way youtube could block your IP, then the answer is yes - use proxy rotation.
Not sure that youtube has written about this something, because then everyone would know how to avoid to be blocked :)
